packages/evolution-generator/src/common/validations.tsx

```diff
@@ -22,7 +22,7 @@ export const requiredValidation: Validations = (value) => {
 };
 
 // Optional question
-export const optionalValidation: Validations = () => [{ validation: false }];
+export const optionalValidation: Validations = () => [];
```

Comment:
Are you sure about that? Empty array for optionalValidation ? Will it works all the time?

Reply:
The `optionalValidation` could also be `undefined` but I'm not sure it would compile. With empty array, it works (see code in `Validation.ts`, it will return true). With the right type, the `{ validation: false }` does not compile as it expects an errorMessage